tests/test_task_download.py

```diff
@@ -1,34 +1,33 @@
 import json
 from pathlib import Path
+from typing import Any, Dict
 
 import pytest
 
 from .tasks import NothingTask
 
-testpath = Path(__file__).parent
 
-
-def get_test_items(name="sentinel2-l2a-j2k-payload"):
-    filename = testpath / "fixtures" / f"{name}.json"
+@pytest.fixture
+def item_collection():
```

Comment:
Looks great.  Part of me would rather `s2_test_collection` for this fixture....but don't hold up the works for that!

Reply:
Yeah, since it was the only fixture used for now I kept it simple -- if we add more fixtures later we can refactor/rename. Thanks for the review!